types/v1/json.go

```diff
@@ -1,43 +1,63 @@
-package types
+package v1
```

Comment:
migrated to under a v1 subpackage, to diff from v2 types

Reply:
new packages look good to me 👍